distribution/zip/jballerina/bin/bal

```diff
@@ -262,6 +262,28 @@ elif [[ $1 == "run" ]] && [[ $2 == *.jar ]]; then # if 1st arg is "run" and 2nd
         -Dballerina.target=jvm \
         -Djava.command=$JAVACMD \
         -jar "${@:2}" # ignores "run"
+
+# running executable(jar) in debug mode
+# if 1st arg is "run", 2nd arg is "--debug", 3rd arg is <debugPort> and 4th arg is executable(jar).
+elif [[ $1 == "run" ]] && [[ $2 == "--debug" ]] && [[ $4 == *.jar ]]; then
```

Comment:
Let's add the required validations for the debug port, before proceeding with the final command.

Reply:
Debug port validation is handled by `agentlib`. When we give an invalid port the following error is thrown:
```
ERROR: transport error 103: invalid port number specified
ERROR: JDWP Transport dt_socket failed to initialize, TRANSPORT_INIT(510)
JDWP exit error AGENT_ERROR_TRANSPORT_INIT(197): No transports initialized [:735]
```

The above behaviour is observed for `bal run --debug <InvalidPort>` and `bal run --debug <InvalidPort> <PATH_TO_JAR>`. Therefore, IMO we don't have to handle this explicitly. WDYT?

Reply:
Yes +1